src/models/stores/user.ts

```diff
@@ -53,6 +53,7 @@ export const UserModel = types
     lastStickyNoteViewTimestamp: types.maybe(types.number)
   })
   .volatile(self => ({
+    waitingForStandaloneAuth: false,
```

Comment:
Should document what this means.

Reply:
This is just a temp variable that will be replaced in the next PR which adds user auth - I've added a `STANDALONE TODO` comment in https://github.com/concord-consortium/collaborative-learning/pull/2486/commits/d37621f9ab623bdcd0908aef2a9b80d58e82921b.